lua/astronvim/utils/ui.lua

```diff
@@ -202,14 +205,17 @@ end
 --- Toggle syntax highlighting and treesitter
 function M.toggle_syntax()
```

Comment:
This Shou take `bufnr` as a parameter just like the other buffer local toggles. This should default to current buffer if none is provided